site/en/docs/handbook/how-to/add-a-blog-post/index.md

```diff
@@ -37,6 +37,19 @@ shortcode snippet, it should look like this:
 
 Paste that into the `hero` field in your YAML frontmatter.
 
+### Social sharing image 
+
+Social sharing image is the visual thumbnail that appears when a post or page 
+is shared on social media platforms like Twitter and Facebook.
+
+Follow the [add media guide](/docs/handbook/how-to/add-media/) to upload your
+image to our CDN. Once you've uploaded the image, copy the path out of the
```

Comment:
I think as https://github.com/GoogleChromeLabs/chrome-gcs-uploader/pull/54#pullrequestreview-1299492930 is almost there, let's add a sentence here like "or use the sharing image generator at ..."